script.js

```diff
@@ -1,4 +1,154 @@
+var user_lose = 0;
+var user_win = 0;
+var user_draw = 0;
+var current_mode = "waiting";
+var user_name = "";
+var myOutputValue = "";
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  if (current_mode == "waiting") {
+    user_name = input;
+    current_mode = "gametime";
+    return (myOutputValue =
+      " hi " + user_name + "<br>" + "it's " + current_mode);
+  } else if (current_mode == "gametime") {
+    return (myOutputValue = playgame(user_name, input));
+  }
+};
+
+var playgame = function (user_name, input) {
+  // Generate a random dice number
+  var randomDiceNumber = rollDice();
+
+  var stone = 1;
+  var scissors = 2;
+  var paper = 3;
+  if (input != "stone" || input != "scissors" || input != "paper") {
+    myOutputValue =
+      "Try again, <br> available input are: scissors , paper or stone";
+  }
+
+  if (input == "stone" && randomDiceNumber == 1) {
+    user_draw = user_draw + 1;
+    myOutputValue =
+      "The computer also chose stone <br> it's a draw <br> Now you can type scissors paper or stone to play another round!" +
+      "<br>" +
+      " so far you draw " +
+      user_draw +
+      " times";
+  }
+  if (input == "scissors" && randomDiceNumber == 2) {
+    user_draw = user_draw + 1;
+    myOutputValue =
+      "The computer also chose scissors <br> it's a draw <br> Now you can type scissors paper or stone to play another round!" +
+      "<br>" +
+      " so far you draw " +
+      user_draw +
+      " times";
+  }
+  if (input == "paper" && randomDiceNumber == 3) {
+    user_draw = user_draw + 1;
+    myOutputValue =
+      "The computer also chose paper <br> it's a draw <br> Now you can type scissors paper or stone to play another round!" +
+      "<br>" +
+      " so far you draw " +
+      user_draw +
+      " times";
+  }
```

Comment:
It would be advisable to convert the dice number into either scissors, paper or stone instead of using numbers in the if statement (as you have done in the later part of your code). That way, instead of having 3 if statements for all the times where they drew, you have have a single if statement. Consider this:

`
 var compChoice = ' '

 if (randomDiceNumber == 1) {
compChoice = stone
}

 else if (randomDiceNumber == 2) {
compChoice = scissors
}

 else if (randomDiceNumber == 3) {
compChoice = paper
}

 if (input == compChoice) {

 user_draw = user_draw + 1;
 myOutputValue =
   `The computer also chose ${compChoice} <br> it's a draw <br> Now you can type scissors paper or stone to play another round! <br> so far you draw ${user_draw} times`
  }

Its just an example (There are multiple ways of doing it) but it does help to cut down a lot of redundant code.